lib/players/human.rb

```diff
@@ -0,0 +1,8 @@
+class Human < Player
+
+  def move(game, board)
+    puts "Enter a number between 1 and 9"
+    input = gets.strip
```

Comment:
You don't need to store your return value in a local variable; just utilize the implicit return value of the expression.
```
def move(board)
  puts "Where would you like to move?"
  gets.chomp
end